src/hub.html

```diff
@@ -26,6 +26,7 @@
         gamma-factor
         networked-scene="adapter: janus; audio: true; debug: true; connectOnLoad: false;"
         physics="gravity: -6; debug: false;"
+        tunneleffect="minRadius: 0.3"
```

Comment:
Since we authored this component, I suggest we change the default `minRadius` value to `0.3` rather than overwrite it just for this instance, now that we know we prefer the value `0.3` instead of `0.25`.